catalog/app/components/SearchResults/SearchResults.js

```diff
@@ -126,16 +126,12 @@ function PackageHeader({ bucket, handle, revision, showBucket }) {
             &nbsp;/{' '}
           </>
         )}
-        <CrumbLink to={urls.bucketPackageTree(bucket, handle, revision)}>
+        <CrumbLink to={urls.bucketPackageTree(bucket, handle, hash)}>
           {handle}
-          {revision !== 'latest' && (
-            <>
-              <M.Box component="span" color="text.hint">
-                @
-              </M.Box>
-              {revision}
-            </>
-          )}
+          <M.Box component="span" color="text.hint">
+            @
+          </M.Box>
+          {R.take(10, hash)}
```

Comment:
Such things as `10` over the code are more maintainable as constants in one place. Much better if we would have some function that responsible for `hash => R.take(10, hash)`. In such a case, we would have a single implementation, that could be changed in one place.